airflow/cli/cli_parser.py

```diff
@@ -626,6 +626,11 @@ def positive_int(value):
     ),
     action="store_true",
     default=False)
+ARG_CONN_FILE_PATH = Arg(
+    ('--file-path',),
```

Comment:
It should be positional argument. 

Example:
```
usage: prog.py [-h] [--sum] N [N ...]

Process some integers.

positional arguments:
 N           an integer for the accumulator

optional arguments:
 -h, --help  show this help message and exit
 --sum       sum the integers (default: find the max)
```
This command have 1 positional argument and 2 keyword argument. 

https://docs.python.org/3/library/argparse.html

Reply:
Make sense, I'll make the change.

Reply:
@mik-laj, I have made the changes. can you review once?